views/auth/login.html

```diff
@@ -10,16 +10,22 @@
         </div>
 
         <h1>Login</h1>
-        {{range .FlashError}}
-          <div class="snackbar snackbar-error">
-            <div class="snackbar-message">
-              <div class="snackbar-close-button-top d-none"></div>
-              <p>{{.}}</p>
-            </div>
-          </div>
-        {{end}}
 
-        <input type="email" name="email" class="form-control mb-4 w-75 mx-auto" placeholder="Email" required autofocus>
+        <div class="mb-4 mx-auto w-75">
+          <div class="row justify-content-center flex-nowrap mx-0">
+            <input type="email" name="email" class="form-control {{if .FlashError}} err-form-control {{end}}" placeholder="Email" required autofocus>
+            {{if .FlashError}} 
+              <div class="icon-warning"></div>
+            {{end}}
+          </div>
+          {{if .FlashError}}
+            <div class="err-message mt-1 ml-1 text-left">
+                {{range .FlashError}} 
```

Comment:
Switched back to the FlashError approach (dynamic messages) and was able to toggle the styling classes with if statement as well.

Below you find the second possible error message in the login page `You must validate your email address`
![image](https://user-images.githubusercontent.com/10324528/63463790-3237d300-c45e-11e9-989b-994452408f47.png)
`
![image](https://user-images.githubusercontent.com/10324528/63463534-a45be800-c45d-11e9-8c25-17a933e9f833.png)